### DISCUSSION TITLE
Sort by relation

### DISCUSSION THREAD

**Discussion by 50l3r** [prisma/prisma#18914] [Q&A]
Hi guys.

There's a way to sort by relation in 2023 or only can sort by _count?

I have this query:

```
this.$prisma.faq.findMany({
  select: {
    id: true,
    sync_id: true,
    created_at: true,
    active: true,
    question: {
      select: {
        translations: {
          select: {
            locale_code: true,
            text: true
          }
        }
      }
    },
    answer: {
      select: {
        translations: {
          select: {
            locale_code: true,
            text: true
          }
        }
      }
    },
    categories: {
      select: {
        id: true,
        sync_id: true,
        title: {
          select: {
            translations: {
              select: {
                locale_code: true,
                text: true
              }
            }
          }
        }
      }
    }
  },
  skip: 0,
  take: 20,
  orderBy: {
    answer: {
      faqAnswers: {
        text: 'desc'
      }
    }
  },
  where: {}
})
```

But i recieve this error:

```
Unknown arg `text` in orderBy.answer.faqAnswers.text for type FaqOrderByRelationAggregateInput. Did you mean `select`? Available args:
type FaqOrderByRelationAggregateInput {
  _count?: SortOrder
}
```

**Comment by ludralph** [comment]
Hi @50l3r 👋 

There is a[ feature request](https://github.com/prisma/prisma/issues/5008) for this and some documented workaround in that issue.

**Reply by 50l3r** [reply]
> Hi @50l3r 👋
> 
> There is a[ feature request](https://github.com/prisma/prisma/issues/5008) for this and some documented workaround in that issue.

Much thanks @ludralph. I had already read the thread but since I have not seen a response since July 15, 2021 I thought that some progress had been made

**Reply by 50l3r** [reply]
> Hi @50l3r 👋
> 
> There is a[ feature request](https://github.com/prisma/prisma/issues/5008) for this and some documented workaround in that issue.

Is it solved already? I see examples but I can't reproduce them.

**Reply by ludralph** [reply]
Hi @50l3r, yes it was solved. I didn't check it was a closed issue 🤦. This is a link to the [documentation](https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting#sort-by-relation) on Sorting by relation.

**Reply by ludralph** [reply]
Hi @50l3r, Did the documentation help in resolving your issue?

**Reply by 50l3r** [reply]
> Hi @50l3r, Did the documentation help in resolving your issue?

I can't test it yet buy i'l keep you updated when i try to fix it :). Much thanks
